src/components/ValueInput/ValueInput.js

```diff
@@ -296,7 +296,7 @@ const ValueInputComponent = ({
         onAssetPress={openAssetSelector}
         labelText={labelText}
         onLabelPress={() => !disabled ? handleUsePercent(100) : undefined}
-        disableAssetSelection={disableAssetChange || assetsOptions.length <= 1}
+        disableAssetSelection={disableAssetChange !== undefined ? disableAssetChange : assetsOptions.length <= 1}
```

Comment:
IMO i would rather modify the logic to autoselect the only asset if there is only one available. Wdyt?

Reply:
Maybe, but then this has to be done on `Exchange` screen and based on any first native asset found it should change initial value accordingly.

Reply:
Hmmm, how about following logic:
1) If there are zero available then display "No assets to select" or similar
2) If there is exactly one asset then automatically select it, trigger onAssetDataChange (as an useEffect)
3) If there are 2+ assets then follow the please select logic.

I guess that this would be optimal from UX perspective, as doing "Please select" when there is only one asset available is an unnecessary step.

wdyt?

Reply:
Thanks for suggestion, but being pragmatic I feel that it's too much of logic added for something that will be refactored soon.